user_manual_de/files/gallery_app.rst

```diff
@@ -84,10 +83,10 @@ funktioniert.
 Grundgerüst
 ^^^^^^^^^^^
 
-Sie sollten einen Kommentar in die Datei einfügen, damit Personen die diese
-Lesen wissen, für was diese bestimmt ist. Kommentare beginnen mit einem #.
+Sie sollten einen Kommentar in die Datei einfügen, damit Personen, die diese
+lesen, wissen für was diese bestimmt ist. Kommentare beginnen mit einem "#".
```

Comment:
Nach `wissen` müsste theoretisch ein weiteres Komma gesetzt werden. Würde den Satz umschreiben:

`Sie sollten einen Kommentar in die Datei einfügen, um deren Nutzen kurz zu erklären.`